articles/user-profile/user-profile-details.md

```diff
@@ -91,7 +91,7 @@ Finally, there is the Authentication API specifically used for authentication fl
 
 In the authentication flows described above, Auth0 returns a set of tokens in lieu of a full User Profile.
 
-One of the returned tokens is the `id_token`, which is a [JSON Web Token](/jwt) (or JWT) that contains User Profile attributes represented in the form of *claims*. These claims are statements about the user, which can be trusted if the consumer of the token can verify its signature (which was generated with the Auth0 app's Client Secret). The app can then use the JWT to securely call other APIs as long as those APIs can verify the JWT's signature and trust and use the contained claims.
+One of the returned tokens is the `id_token`, which is a [JSON Web Token](/jwt) (or JWT) that contains User Profile attributes represented in the form of *claims*. These claims are statements about the user, which can be trusted if the consumer of the token can verify its signature (which was generated with the Auth0 app's Client Secret). The app can then decode the JWT and get the user information contained in its payload, like the user's name, email, and so forth, typically used for UI display.
```

Comment:
The ID token can also be signed with a private key and verified with a public key - some clients such as SPAs are not capable of storing secrets so they don't have a client secret at all.